cocos/ui/edit-box/EditBox-mac.mm

```diff
@@ -88,6 +88,9 @@ - (void)controlTextDidChange:(NSNotification *)notification {
     NSTextField *textField = [notification object];
     callJSFunc("input", [textField.stringValue UTF8String]);
 }
+- (void)controlTextDidEndEditing:(NSNotification *)obj {
+    cc::EditBox::complete();
+}
```

Comment:
controlTextDidEndEditing would be called when textField loses focus or the key return is pressed

Reply:
Good to know.